fuse/src/reply.rs

```diff
@@ -245,14 +254,14 @@ impl Reply for ReplyEntry {
 
 impl ReplyEntry {
     /// Reply to a request with the given entry
-    pub fn entry(self, ttl: &Timespec, attr: &FileAttr, generation: u64) {
+    pub fn entry(self, ttl: &Duration, attr: &FileAttr, generation: u64) {
```

Comment:
Am I correct in thinking these `ttl`'s should be `Duration`s?
Or is the `ttl` meant to be a timestamp?

Reply:
Changing this from a `Duration` to `SystemTime` has fixed the tests.